src/tokenizer.rs

```diff
@@ -102,6 +103,24 @@ pub enum Token {
     RBrace,
     /// Right Arrow `=>`
     RArrow,
+    /// Sharp `#` use for PostgreSQL Bitwise XOR operator
+    Sharp,
+    /// Tilde `~` use for PostgreSQL Bitwise NOT operator
+    Tilde,
+    /// Bitwise left operator `<<` use for PostgreSQL
+    ShiftLeft,
+    /// Bitwise right operator `>>` use for PostgreSQL
+    ShiftRight,
+    /// Exclamation Mark `!` use for PostgreSQL factorial operator
+    ExclamationMark,
+    /// Exclamation Mark `!!` use for PostgreSQL prefix factorial operator
+    DoubleExclamationMark,
+    /// Ampersat `@` use for PostgreSQL abs operator
+    Ampersat,
+    /// PostgreSQL square root math operator
+    SquareRoot,
+    /// PostgreSQL cube root math operator
+    CubeRoot,
```

Comment:
Please mention that these are `|/` and `||/` respectively. Consider using the same names as for the UnaryOperator. I'm in favor of PGSquareRoot and PGCubeRoot, as these are probably very specific to postgres, and CubeRoot is more obvious than "cbrt".